nixos/doc/manual/release-notes/rl-1809.xml

```diff
@@ -77,6 +77,8 @@ following incompatible changes:</para>
 <itemizedlist>
   <listitem>
     <para>
+      <literal>dockerTools.pullImage</literal> relies on image digest
+      instead of image tag to download the image.
```

Comment:
Ok this was already done. It should also mention that hashes will change.

Reply:
Done